recipes/qcfractal/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "qcfractal" %}
+{% set version = "0.5.0rc1" %}
+{% set sha256 = "0c08b7c3b7097f75be86d77092a5dfdf31aa3e6e1818e05cddc25002e648e02a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Set-setup.py-to-zip_safe-False-for-Windows-testing.patch
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+
+  run:
+    - python >=3.6
+    - numpy
+    - pandas
+    - mongodb
+    - tornado
+    - requests
+    - bcrypt
+    - cryptography
+    - pydantic
+    - mongoengine
+    - qcengine >=0.5.1
+    - qcelemental >=0.2.4
+
+  run_constrained:
```

Comment:
I think there is no need for these to be specified unless there is a version requirement.

Reply:
Ahhh. So this is only if the optional dep has a version constraint that must be obeyed. That makes sense.

Reply:
That is what I understood this block to indicate from the docs, but its also the only block which lets you do any kind of specification of "Optional packages you can install, but don't have to, to get fancy features."

I'll remove the block since there are not version specs needed and therefore this contributes nothing to the recipe. Stronger docs on our end it is then.

Reply:
Yup. It can also be abused to define conflicting packages by specifying a nonsense version like `conflicting_package ==99999999`.